powerstore/resource_snapshot.go

```diff
@@ -258,6 +237,65 @@ func (r *resourceVolumeSnapshot) Read(ctx context.Context, req resource.ReadRequ
 
 // Update - updates volume snapshot resource
 func (r *resourceVolumeSnapshot) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	log.Printf("Started Update")
```

Comment:
Are we supporting the updation of the expiration timestamp/description?

Reply:
yes we are supporting both

Reply:
Can you please point out the comparison logic?